spec/features/api_reference_spec.rb

```diff
@@ -55,5 +55,7 @@ def then_there_is_correct_api_schema_content
     expect(page).to have_css("h3#schema-pet", text: "Pet")
     # Schema parameters
     expect(page).to have_css("table", text: /\b(tag )\b/)
+    # Check that the "required" column is true for the `id` attribute
+    expect(page).to have_css("table.schema-pet td:nth(3)", text: "true")
```

Comment:
Would be great if this test were a bit more self-documenting - I wonder if we can dynamically infer the correct column number? I appreciate that introduces some complexity to the test, so may not be an improvement.

Reply:
I don't really want to edit someone else's code there, nor particularly going into the world of testing for the API stuff. It seems a big can of worms as there is a lack of testing for it all.